examples/README.md

```diff
@@ -0,0 +1,68 @@
+# Connector Examples
+
+This directory contains examples of connectors that can be used as a starting point for developing your own connectors.
+
+## `simple-connector/` 
+
+The contents of this directory demonstrate the basic usage of the `inorbit-connector` library. It can be run with the following command:
+
+```bash
+python simple-connector/simple_connector.py
```

Comment:
Remove this. All the steps are described below, so suggesting this at this point can be misleading